user/views.py

```diff
@@ -0,0 +1,53 @@
+import json, re
+import bcrypt
+import jwt
+
+from django.http      import JsonResponse
+from django.views     import View
+from django.db.models import Q
+
+from .models     import User
+from my_settings import SECRET_KEY, ALGORITHM
+
+class SignUpView(View):
+    def post(self,request):
+        try:
+            data         = json.loads(request.body)
+            email        = data['email']
+            password     = data['password']
+            name         = data['name']
+            phone_number = data['phone_number']
+            nickname     = data.get('nickname', None)
+
+            if User.objects.filter(Q(email=email) | Q(name=name) | Q(phone_number=phone_number)).exists():
+                return JsonResponse({'message':' 이미 가입되어 있습니다.'}, status=400)
```

Comment:
저희가 status 코드 이외에 message를 주는 이유는한글로 클라인트에게 설명하는것이 아니고 같은 status 코드에 여러 상황이 있기 때문에 그 상황을 프론트엔드에게 알려주기 위해서 입니다. 따라서 최소한으로 메시지를 전달하는 것이 좋습니다.

다른 return 메시지들도 확인해보고 수정해 주세요.
 
```suggestion
                return JsonResponse({'message' : 'DUPLICATED_INFORMATION'}, status=400)
```

Reply:
넹 확인했습니다-! 전부 수정하겠습니다